docs/data/sdks/android-kotlin/index.md

```diff
@@ -43,7 +43,7 @@ The SDK internally uses a few Java 8 language APIs through desugaring. Make sure
 
 ### Step 3: Initialization
 
-Before you can instrument, you must initialize the SDK using the API key for your Amplitude project. You can use the Android SDK anywhere after it's initialized in an Android application.
+Before you can instrument, you must initialize the SDK using the API key for your Amplitude project. We recommend doing the initialization in the Main Activity, which will never get destroyed. Or the Application class if you have one. Then You can use the Android SDK anywhere after it's initialized in an Android application.
```

Comment:
Minor adjustments on the sentence.
```suggestion
Before you can instrument, you must initialize the SDK using the API key for your Amplitude project. We recommend doing the initialization in the main activity, which will never get destroyed, or in the `Application` class if you have one. After it's initialized, you can use the Android SDK anywhere in your Android application.
```

Reply:
The default android project template usually has `MainActivity` class and `MainApplication`. 
Maybe we can use `MainActivity` or `Application`?